peachjam/static/stylesheets/components/_document-content.scss

```diff
@@ -73,6 +73,9 @@
   }
 
   .content-and-enrichments__inner {
+    max-width: 80em;
```

Comment:
I chose 80em because 55em looked to squashed

Reply:
Ah, this is simpler than I thought. It keeps the content with the gutter, otherwise on wide views the gutter will be far away. Nice.

Since the content is 80% of the full width, 70 * 0.8 = 56, so let's go with 70em.

Additionally, pls  can you add `font-size: 15px` to `.content.content__html` above.

Reply:
done